src/OrbitCaptureClient/ApiEventProcessor.cpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitCaptureClient/ApiEventProcessor.h"
+
+#include "OrbitBase/Logging.h"
+
+using orbit_client_protos::TimerInfo;
+using orbit_grpc_protos::ApiEvent;
+
+ApiEventProcessor::ApiEventProcessor(CaptureListener* listener) : capture_listener_(listener) {}
```

Comment:
`CHECK` that `listener` is not `nullptr`.

Reply:
Done.